addons/html_editor/static/src/core/format_plugin.js

```diff
@@ -100,12 +100,12 @@ export class FormatPlugin extends Plugin {
                 },
                 icon: "fa-eraser",
                 title: _t("Remove Format"),
-                hasFormat: hasFormat(p),
+                hasFormat: hasFormat(this),
             },
         ],
         arrows_should_skip: (ev, char, lastSkipped) => char === "\u200b",
-        onBeforeInput: { handler: p.onBeforeInput.bind(p), sequence: 60 },
-    });
+        onBeforeInput: withSequence(20, this.onBeforeInput.bind(this)),
```

Comment:
It looks like a big number change, might have problems with other plugins' `onBeforeInput` :thinking: